tests/test_flows.py

```diff
@@ -852,31 +852,33 @@ def test_client_capability_query(self):
 
         # first check capability query fails on unconnected station
         debug("Send client capability query for unconnected STA")
-        mid1 = env.controller.dev_send_1905(env.agents[0].mac, 0x8009,
-                                            tlv(0x90, 0x000C, '{} {}'.format(
-                                                env.agents[0].radios[0].mac, sta1.mac)))
+        mid_for_error = env.controller.dev_send_1905(env.agents[0].mac, 0x8009,
```

Comment:
As noted elsewhere, this is not needed if we simply split the test into two tests.

Also, changes that are a result of review should typically be done as fixups. So in this case, do an interactive rebase and edit the commit that introduces mid1 and fix it there.